package.json

```diff
@@ -46,19 +46,22 @@
     "trailingComma": "es5"
   },
   "devDependencies": {
+    "@testing-library/react": "^9.1.3",
     "@types/jest": "^24.0.18",
     "@types/react": "^16.8.25",
     "@types/react-dom": "^16.8.5",
     "husky": "^3.0.2",
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "tsdx": "^0.7.2",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "tsdx": "^0.9.0",
     "tslib": "^1.10.0",
     "typescript": "^3.5.3"
   },
   "dependencies": {
+    "@types/pubnub": "^4.0.6",
+    "pubnub": "^4.25.1",
```

Comment:
I think pubnub might be a peerDependency (and a devDependency) on this package.  It's critical that PubNub not end up in the built package for this library, but rather be supplied by the person using the library.  This library could be used with a wide variety of versions of the PubNub SDK.

Reply:
Yeah, that makes sense.